src/pfe/portal/modules/ExtensionList.js

```diff
@@ -42,7 +42,7 @@ module.exports = class ExtensionList {
           await extension.initialise();
           this.add(extension);
           if (extension.templates)
-            templates.addRepository(extension.templates, extension.description);
+            await templates.addRepository(extension.templates, extension.description);
```

Comment:
I know the method was already this way, but could you put a try/catch inside the for loop, so that even if 1 extension fails to load at any of the `await` statements, it won't crash out of the loop and other extensions can still load?